firestore/cloud-client/snippets.py

```diff
@@ -815,3 +816,63 @@ def delete_collection(coll_ref, batch_size):
     # [END delete_full_collection]
 
     delete_collection(db.collection(u'cities'), 10)
+
+
+@pytest.mark.skip(reason="Dependant on a composite index being created,"
```

Comment:
Why does this need to be here? Aren't the tests in snippet_tests.py?

This should snippet should also probably have tests on it.

Reply:
You're right, I moved it over to the tests file and added an assertion there.